.github/workflows/rust.yml

```diff
@@ -67,14 +67,21 @@ jobs:
           toolchain: nightly
           target: wasm32-unknown-unknown
           override: true
+      - name: Add component llvm-tools-preview
+        run: rustup component add llvm-tools-preview --toolchain nightly-x86_64-unknown-linux-gnu
       - name: Rust Cache
         uses: Swatinem/rust-cache@v1.3.0
-      - name: Run Tarpaulin
-        uses: actions-rs/tarpaulin@v0.1
-        with:
-          args: '--locked --jobs 16 --timeout 600 --skip-clean -- --test-threads 16'
+      - name: Run cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Generate code coverage
+        run: cargo llvm-cov --workspace --lcov --output-path lcov.info
+        # with:
+        #   args: '--locked --jobs 16 --timeout 600 --skip-clean -- --test-threads 16'
```

Comment:
`cargo llvm-cov` takes `--jobs` parameter. We might as well set it to 16.

Reply:
Oh wait, it defaults to the number of available cpus. Yeah, let's leave it then.

Reply:
Let's just remove the commented-out old arguments since we're not using them anymore.